src/components/History/HistoryPage.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import {UseHistory} from "../../context/HistoryProvider";
+import {UseLiked} from "../../context/LikedProvider";
+import "../../styles/root.css";
+import {NormalCard} from "../Cards/NormalCard";
+
+export function HistoryPage() {
+  const {HistoryVideos, removeFromHistory} = UseHistory();
+
+  return (
+    <>
```

Comment:
You can remove the fragments as it is not required here.

Reply:
Okay, I'll do that, Thank you Vrinda for PR review.